src/script/components/panel/UserDetails.tsx

```diff
@@ -62,8 +64,8 @@ const UserDetails: React.FC<UserDetailsProps> = ({
     amplify.publish(WebAppEvents.USER.UPDATE, participant.qualifiedId);
   }, [participant]);
 
-  const isFederated = participant.isFederated;
-  const isGuest = !isFederated && participant.isGuest();
+  const isFederated = user.isFederated;
```

Comment:
Did you test this code? This is breaking since `user` doesn't have the `isFederated` property